x-pack/test/defend_workflows_cypress/runner.ts

```diff
@@ -10,6 +10,56 @@ import Url from 'url';
 import { withProcRunner } from '@kbn/dev-proc-runner';
 
 import { FtrProviderContext } from './ftr_provider_context';
+import { AgentManager } from './agent';
+import { AgentManagerParams, FleetManager } from './fleet_server';
+
+async function withFleetAgent(
+  { getService }: FtrProviderContext,
+  runner: (runnerEnv: Record<string, string>) => Promise<void>
+) {
+  const log = getService('log');
+  const config = getService('config');
+
+  const esHost = Url.format(config.get('servers.elasticsearch'));
+  const params: AgentManagerParams = {
+    user: config.get('servers.elasticsearch.username'),
+    password: config.get('servers.elasticsearch.password'),
+    esHost,
+    esPort: config.get('servers.elasticsearch.port'),
+    kibanaUrl: Url.format({
+      protocol: config.get('servers.kibana.protocol'),
+      hostname: config.get('servers.kibana.hostname'),
+      port: config.get('servers.kibana.port'),
+    }),
+  };
+  const requestOptions = {
+    headers: {
+      'kbn-xsrf': 'kibana',
+    },
+    auth: {
+      username: params.user,
+      password: params.password,
+    },
+  };
+  const fleetManager = new FleetManager(params, log, requestOptions);
+  const agentManager = new AgentManager(params, log, requestOptions);
+
+  // Since the managers will create uncaughtException event handlers we need to exit manually
+  process.on('uncaughtException', (err) => {
+    // eslint-disable-next-line no-console
+    console.error('Encountered error; exiting after cleanup.', err);
```

Comment:
Suggestion: remove `console.error` (and the eslint rule bypass) and use `log` which is defined in line 20.